xbmc/addons/IAddon.h

```diff
@@ -92,6 +92,7 @@ namespace ADDON
     virtual bool GetSettingString(const std::string& key, std::string& value) = 0;
     virtual CAddonSettings* GetSettings() const =0;
     virtual const ADDONDEPS &GetDeps() const =0;
+    virtual int NoDeps(std::map<std::string,std::string>& added) const = 0;
```

Comment:
renamed, doxified and wrapped. due to the IAddon mess i need the recursive function as a virtual as well, public at that. sigh. oh, and yes i know i could have used a cast, but this way it's transparent when IAddon dies and diff would simply be moving the CAddon method to protected and removing override (right @AlwinEsch ?)